tests/corobo_test_case.py

```diff
@@ -0,0 +1,41 @@
+from errbot.backends.test import FullStackTest
+from errbot.plugin_info import PluginInfo
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
+
+import logging
+
+
+class CoroboTestCase(FullStackTest):
+
+    def setUp(self,
+              extra_plugin_dir=None,
```

Comment:
I think you can remove all these args.

They are not being used, and unittest and pytest do not use them.